.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+on: [push, pull_request]
+env:
+  PYTEST_ADDOPTS: -vv --cov-append
+
+jobs:
+  test:
+    name: ${{ matrix.python }} ${{ matrix.target }}
+    strategy:
+      matrix:
+        python: [2.7, 3.5, 3.6, 3.7, 3.8, 3.9, 'pypy2', 'pypy3']
+        target: ['tests']
+        include:
+          - python: 3.6
+            target: 'checkqa'
+    runs-on: ubuntu-20.04
+    env:
+      CI_TARGET: ${{ matrix.target }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        id: python
+        with:
+          python-version: ${{ matrix.python }}
+
+      - if: matrix.target == 'checkqa'
+        run: printf 'TOXENV=checkqa,docs\n' >> $GITHUB_ENV
+
+      - if: matrix.target == 'tests'
+        run: |
+          mkdir "$GITHUB_WORKSPACE/_neovim"
+          wget -q -O - https://github.com/neovim/neovim/releases/download/nightly/nvim-linux64.tar.gz | tar xzf - --strip-components=1 -C "$GITHUB_WORKSPACE/_neovim"
+          printf 'PATH=%s:%s\n' "$GITHUB_WORKSPACE/_neovim/bin" "$PATH" | tee -a $GITHUB_ENV
+          printf 'VIMRUNTIME=%s\n' "$GITHUB_WORKSPACE/_neovim/share/nvim/runtime" | tee -a $GITHUB_ENV
```

Comment:
This type of thing might be worth creating an actual GitHub action (e.g., neovim/action) for people to use.

Reply:
yes :)